Demo/HYPFormsTests/HYPFormsCollectionViewDataSourceTests.m

```diff
@@ -82,6 +82,48 @@ - (void)testIndexInForms
     [self.dataSource processTarget:[HYPFormTarget showFieldTargetWithID:@"last_name"]];
 }
 
+- (void)testEnableAndDisableTargets
+{
+    HYPFormField *targetField = [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
+
+    XCTAssertFalse(targetField.disabled);
+
+    HYPFormTarget *disableTarget = [HYPFormTarget disableFieldTargetWithID:@"display_name"];
+
+    [self.dataSource processTarget:disableTarget];
+
+    XCTAssertTrue(targetField.disabled);
+
+    HYPFormTarget *enableTarget = [HYPFormTarget enableFieldTargetWithID:@"display_name"];
+
+    [self.dataSource processTargets:@[enableTarget]];
+
+    XCTAssertFalse(targetField.disabled);
+}
+
+- (void)testUpdatingTargetValue
+{
+    HYPFormField *targetField = [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
+    XCTAssertEqualObjects(targetField.fieldValue, @" ");
+
+    HYPFormTarget *updateTarget = [[HYPFormTarget alloc] initWithDictionary:@{
+                                                                              @"id" : @"display_name",
+                                                                              @"type" : @"field",
+                                                                              @"action" : @"update",
+                                                                              @"target_value": @"John Hyperseed"}];
+
+    [self.dataSource processTarget:updateTarget];
+
+    XCTAssertEqualObjects(targetField.fieldValue, @"John Hyperseed");
+}
+
+- (void)testDefaultValue
+{
+    HYPFormField *usernameField = [self.manager fieldWithID:@"username" includingHiddenFields:YES];
+
+    XCTAssertNotNil(usernameField.fieldValue);
```

Comment:
Well, it does but the test is funky as it relies on an action in forms.json.
Good that you spotted this because changing the sample would actually break the test.